docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md

```diff
@@ -0,0 +1,194 @@
+---
+id: docusign-source
+title: DocuSign
+sidebar_label: DocuSign
+description: This document explains how to collect customer event data from the DocuSign and send it to Sumo Logic.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+DocuSign pioneered the development of e-signature technology, and today DocuSign helps organizations connect and automate how they prepare, sign, act on, and manage agreements. As part of the DocuSign Agreement Cloud, DocuSign offers eSignature: the world's #1 way to sign electronically on practically any device, from almost anywhere, at any time.
```

Comment:
Tone down the marketing language (our focus should be what the Sumo Logic app can do for customers)

```suggestion
DocuSign pioneered the development of e-signature technology, and today DocuSign helps organizations connect and automate how they prepare, sign, act on, and manage agreements. As part of the DocuSign Agreement Cloud, DocuSign offers eSignature, allowing you to sign documents electronically from most devices.
```